src/components/posts/PostItem.js

```diff
@@ -0,0 +1,24 @@
+const PostItem = ({ post }) => {
+  return (
+    <li className="post-item">
```

Comment:
Can we use single quotes for these please? So it's inline with the other components in the app